packages/client/types/job.ts

```diff
@@ -8,22 +8,34 @@ import {
 
 export type Job = PrimaryKey & {
   company_id: string;
-  title: string;
-  description: string;
-  position?: string;
-  opportunity?: string;
+  jobtitle: string;
+  jobdescription: string;
+  jobposition: string;
+  jobtype: string;
+  jobcompensation: string;
+  jobmax: string;
+  jobmin: string;
+  jobequity: string;
+  joblocation: string;
+  jobcontact: string;
```

Comment:
We can omit keeping these type definitions up-to-date since we are using the automatic type generator from Supabase. This comment also applies to other type definitions that were involved in changes.

See this PR for context #116